src/NHibernate.Test/Criteria/AddNumberProjection.cs

```diff
@@ -35,7 +35,7 @@ public override SqlString ToSqlString(ICriteria criteria, int position, ICriteri
 				.Add(" + ")
 				.Add(criteriaQuery.NewQueryParameter(typedValue).Single())
 				.Add(") as ")
-				.Add(GetColumnAliases(0, criteria, criteriaQuery)[0])
+				.Add(GetColumnAlias(0, 0 ))
```

Comment:
Why do you need second 0 here? It is almost always `0`.

Reply:
To allow `GetColumnAlias` method to change multicolumn projection aliases in user overrides. With you suggestion to get rid of virtual - it's not useful so I removed it.